### ISSUE TITLE
A prison system

### ISSUE THREAD

**Issue by samantha-ad** [Linesmerrill/police-cad#684]
a prison system where a person has a tag next to their name when it is run that states that they are incarcerated.

**Implementation Details**
when a person's name is run the officer has the option to arrest a person and set the amount of time for how long they are arrested in min and sec

**Comment by Linesmerrill** [comment]
Yeah this is a great suggestion! Let me pin this so our devs can take a look! Thanks
